NF.jsonld

```diff
@@ -7675,6 +7781,28 @@
     "rdfs:comment" : "Characterization of the manifestation of Dermal neurofibromas.",
     "@type" : "rdfs:Class",
     "sms:displayName" : "dermalNeurofibromas"
+  }, {
+    "@id" : "bts:plateWell",
+    "@type" : "rdfs:Class",
+    "rdfs:comment" : "User-specified identifier for the specific well of the plate used to prepare the sample for analysis.",
+    "rdfs:label" : "plateWell",
+    "rdfs:subClassOf" : [ ],
+    "schema:isPartOf" : {
+      "@id" : "http://schema.biothings.io/"
+    },
+    "sms:displayName" : "plateWell",
+    "sms:required" : "sms:false"
+  }, {
+    "@id" : "bts:chipID",
```

Comment:
@anngvu - isn't this the same as `platform`? Could we consolidate with that concept (and perhaps also we could specify GEO platform ids for arrays....? :) (